nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/chronicle/ChronicleRetentionDuration.java

```diff
@@ -72,21 +84,20 @@ public int getRetentionCycles() {
 
     protected RollCycle decodeRollCycle(String retentionDuration) {
         if (retentionDuration == null || retentionDuration.isEmpty()) {
-            return RollCycles.DAILY;
+            return DAILY;
         }
         String rollingPeriod = retentionDuration.substring(retentionDuration.length() - 1);
         switch (rollingPeriod) {
         case SECOND_ROLLING_PERIOD:
             return RollCycles.TEST_SECONDLY;
```

Comment:
Can use the static import too. And I assume you can then remove the `RollCycles` import.